### ISSUE TITLE
Default visibility when replying to post

### ISSUE THREAD

**Issue by amz71** [mastodon/mastodon#21745]
### Pitch

Too much to read. Current default visibility "public" should be moved to "not listed" when replying to other people's posts.

### Motivation

Too many posts, too fast federated timelines.

**Comment by tithonium** [comment]
Letting users choose that for themselves would be okay, I think, but not everybody would want that as the default (I wouldn't, for example).

**Comment by amz71** [comment]
Yes, default to "not listed" and let them chose to set it to public 😎

**Comment by nexxai** [comment]
100%.  

There's already a setting in preferences for default post visilbility so it would be great if a second setting could be added for replies.
